finans/rapport_includes/moms_transaktioner.php

```
@@ -0,0 +1,356 @@
+    // --- VAT code filter dropdown ---
+    $all_codes = [];
+    $q = db_select("SELECT DISTINCT kp.moms, g.beskrivelse FROM kontoplan kp"
+                 . " LEFT JOIN (SELECT DISTINCT ON (art,kodenr) art, kodenr, beskrivelse FROM grupper"
+                 . "   WHERE art IN ('SM','KM','YM','EM') ORDER BY art, kodenr, fiscal_year DESC NULLS LAST) g"
+                 . " ON g.art = UPPER(SUBSTRING(kp.moms FROM 1 FOR 1)) || 'M'"
+                 . " AND CAST(g.kodenr AS TEXT) = SUBSTRING(kp.moms FROM 2)"
+                 . " WHERE kp.regnskabsaar = '$regnaar' AND kp.moms IS NOT NULL AND kp.moms != ''"
+                 . " ORDER BY kp.moms", __FILE__." linje ".__LINE__);
+    while ($r = db_fetch_array($q)) $all_codes[$r['moms']] = $r['beskrivelse'];
+
+    $filter_url_base = $back_url . ($filter_kode !== '' ? "&filter_momskode=$filter_kode" : '');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$filter_url_base'. (undefined)

(UnusedLocalVariable)